src/resources/js/embedded-map.js

```diff
@@ -3,13 +3,14 @@
  */
 if ( "function" === typeof jQuery ) jQuery( document ).ready( function( $ ) {
 	var mapHolder,
-		position,
-		streetAddress,
-		venueCoords,
-		venueTitle;
+	    position,
+	    venueObject,
+	    venueAddress,
+	    venueCoords,
+	    venueTitle;
 
 	// The tribeEventsSingleMap object must be accessible (as it contains the venue address data etc)
-	if ("undefined" === typeof tribeEventsSingleMap ) return;
+	if ( "undefined" === typeof tribeEventsSingleMap ) return;
```

Comment:
I guess we could - they're not newly introduced with this commit though and our [JS coding standards](https://github.com/moderntribe/the-events-calendar/wiki/Javascript-Coding-Style) don't currently specify we should prefer either ... is there a strong reason to prefer single quotes here?